src/pages/home/report/ReportActionCompose/index.js

```diff
@@ -524,8 +523,7 @@ class ReportActionCompose extends React.Component {
      * @param {String} emoji
      */
     addEmojiToTextBox(emoji) {
```

Comment:
Should remove this function now that it's only calling `replaceSelectionWithInput`?

Reply:
I see the PR that introduces `replaceSelectionWithInput ` will be reverted here https://github.com/Expensify/App/pull/17475, so I think removing it would be a waste of work.